<map id="django.utils.unittest.case.TestCase.id" name="django.utils.unittest.case.TestCase.id">
<area shape="rect" id="node2" href="$db/da8/classdjango_1_1utils_1_1unittest_1_1suite_1_1___error_holder.html#aeae5b5ec7f4966b6ab217eed7de2571f" title="django.utils.unittest.suite.\l_ErrorHolder.__str__" alt="" coords="253,5,424,47"/>
<area shape="rect" id="node3" href="$dd/d91/classdjango_1_1contrib_1_1sitemaps_1_1tests_1_1base_1_1_test_model.html#a6f320acea39496650160136996455131" title="django.contrib.sitemaps.tests.base.\lTestModel.get_absolute_url" alt="" coords="224,71,453,112"/>
<area shape="rect" id="node4" href="$dc/db0/classdjango_1_1contrib_1_1auth_1_1models_1_1_abstract_user.html#a4eea92addbb28b562d768d3590615a1e" title="django.contrib.auth.models.\lAbstractUser.get_profile" alt="" coords="247,136,430,177"/>
</map>
